collect_app/src/androidTest/java/org/odk/collect/android/support/ResetStateRule.java

```diff
@@ -93,4 +94,9 @@ private void clearPrefs() {
         settingsProvider.getAdminSettings().setDefaultForAllSettingsWithoutValues();
         settingsProvider.getMetaSettings().clear();
     }
+
+    private void clearProjects() {
+        DaggerUtils.getComponent(InstrumentationRegistry.getInstrumentation().getTargetContext()).projectsRepository().deleteAll();
+        DaggerUtils.getComponent(InstrumentationRegistry.getInstrumentation().getTargetContext()).projectImporter().importDemoProject();
```

Comment:
Do we need the `deleteAll` given meta prefs is cleared or do you think it's better to be explicit here (and not assume the projects are stored in prefs)? I'm guessing we need to do the import again because `ApplicationInitializer` won't run? Maybe we need to think about running it here. That's an improvement we can make down the road though.

Reply:
Right, `deleteAll()` is not needed here so I removed it. When it comes to `ApplicationInitializer` it is called here but it imports the default project only if it's not fresh install but in tests, we may have different scenarios so I added it here to make sure it works well. I could also override `AppStateProvider` to return false always for `isFreshInstall()` but it might be bad for other tests that we want to add in the future.